CHANGES.rst

```diff
@@ -11,6 +11,10 @@ https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
 ------------------
 
 - Update to newest compatible versions of dependencies.
+- Fix history page for classes modifying instances in ``__setstate__``,
```

Comment:
```suggestion

- Fix history page for classes modifying instances in ``__setstate__``,
```

Reply:
Thanks, I applied this locally and push-forced